packages/babel-helper-fixtures/src/index.js

```diff
@@ -125,6 +128,22 @@ export default function get(entryLoc): Array<Suite> {
         },
       };
 
+      // If there's node requirement, check it before pushing task
+      if (taskOpts.minNodeVersion) {
+        const minimumVersion = semver.clean(taskOpts.minNodeVersion);
+
+        if (minimumVersion == null) {
+          throw new Error(`'minNodeVersion' has invalid semver format: ${taskOpts.minNodeVersion}`);
+        }
+
+        if (semver.lt(nodeVersion, minimumVersion)) {
+          return;
+        }
+
+        // Delete to avoid option validation error
+        delete taskOpts.minNodeVersion;
```

Comment:
If not, the test will throw `Unknown option: base.minNodeVersion.` I suppose we can update `OptionManager` to support this option, but given it's only internally used, I'm not sure if we should do that.